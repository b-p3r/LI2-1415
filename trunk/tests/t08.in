5 5
TASOR
IMAUD
NECIE
RISTS
ENROI
