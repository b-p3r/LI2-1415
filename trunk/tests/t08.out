A
ACEITA
ACEITAM
ACEITAS
ACESO
ACESOS
ACIDO
ACIDOS
ACTO
ACTOR
ACTOS
ACUDE
ACUDES
ACUDI
ACUSA
ACUSAM
ACUSO
AI
AIS
AMA
AMAINE
AMAIS
AMAS
AME
AMEI
AMEIA
AMEIAS
AMEN
AMERICA
AMERICAS
AMIEIRO
AMIEIROS
AMNESIA
AMNESIAS
AMNISTIA
AMNISTIAS
AMNISTIE
AO
AOS
AS
ASA
ASMA
ASO
AUDITOR
CA
CAEM
CAES
CAI
CAIDO
CAIDOS
CAIS
CAIU
CAMA
CAMAS
CAO
CAOS
CASA
CASAM
CASO
CASOU
CAUSA
CAUSAM
CAUSO
CE
CEIA
CEIAS
CEM
CERNE
CERNIR
CESTO
CESTOS
CINEMA
CINEMAS
CISNE
CITE
CITES
CITO
CU
CUIDE
CUIDO
CURDO
CURDOS
CURO
CUS
DE
DEI
DEITO
DES
DESISTI
DESISTO
DESTRO
DESTROI
DESTROS
DEU
DEUS
DEUSA
DIA
DIAS
DICA
DICAS
DISCA
DISCERNE
DISCERNIR
DISTO
DITE
DITO
DITOS
DO
DOA
DOAM
DOAS
DOR
DOS
DOSA
DOU
DRUSA
DRUSO
DUAIS
DUAS
DUCAIS
DUCTEIS
DUETO
DUETOS
DUO
DURO
DUROS
E
EDICAO
EDIS
EDITO
EDITOR
EDITOS
EDUCA
EDUCAM
EIA
EIS
EITO
EM
EMITA
ENROSCA
ENSINE
ENSINEI
ENSINEM
ERECTEIS
ERECTO
ERECTOS
EREMITA
EREMITAS
ES
ESCAMA
ESCAMAS
ESCUDE
ESCUDO
ESCUDOS
ESCURO
ESCUROS
ESCUSA
ESCUSAM
ESCUSO
ESTE
ESTES
ESTICA
ESTICAM
ESTICAO
ESTIO
ETICA
ETICAS
EU
EURO
EUROS
IA
IAM
IAO
IAS
ICA
ICAM
ICE
ICEM
IDE
IDES
IDO
IDOS
IDOSA
IMA
INICIA
INICIAM
INICIAS
INICIE
INSECTO
INSECTOS
INSERE
INSISTE
INSISTI
INSISTO
IR
IRE
IREI
IREM
IRES
IS
ISCA
ISCAS
ISTO
MA
MACE
MACUDO
MAE
MAES
MAESTRO
MAESTROS
MAIS
MAO
MAOS
MAS
MAU
MAUS
ME
MECA
MECAS
MEIA
MEIAS
MENIR
MES
MESTICA
MESTICAS
MI
MIA
MIAS
MIEI
MINE
MINEIRO
MINEIROS
MINES
MINI
MINISTRO
MINISTROS
NEM
NESCIA
NESTE
NESTES
NICAS
NICE
NINE
NISTO
O
ODE
ODEIA
ODEIAM
ODES
OS
OU
OUCA
OUCAM
OUSA
OUSAM
RE
REACTOR
REAIS
RECAI
RECAIDO
RECAIU
RECITE
RECTO
RECTOS
RECUA
RECUAM
RECUE
RECUEI
RECUO
RECUOS
RECUSA
RECUSAM
RECUSO
REI
REINE
REINEM
REINICIA
REINICIAM
REINICIE
REIS
REMA
RES
RESIDE
RESIDO
RESIDUO
RESIDUOS
RESISTE
RESISTI
RESISTO
RESTE
RESTIA
RESTIAS
RESTO
RESTOS
RESTRICAO
RI
RICA
RICAS
RIEM
RINS
RIR
RIREM
RIRES
RIS
RISCA
RISCAM
RISCAS
RISO
RISOS
RISTE
RO
ROA
ROAM
RODE
RODEI
RODEIA
RODEIAM
ROI
ROSA
ROSCA
ROSCAS
ROSE
ROSEA
ROSEAS
ROUCA
ROUCAS
RUA
RUAS
RUCA
RUCAS
RUDE
RUDES
RUI
RUIA
RUIAM
RUIDO
RUIDOS
RUIDOSA
SA
SAEM
SAI
SAIDO
SAIS
SAISTE
SAIU
SAM
SAO
SAUDE
SAUDEI
SAUDES
SAUDO
SE
SECA
SECAM
SECAS
SECTOR
SEDUCAO
SEI
SEIS
SEITA
SEITAS
SEM
SEMITA
SEMITAS
SER
SEREI
SERIE
SEU
SEUS
SI
SIAMES
SIDO
SIDOS
SIR
SISO
SITE
SITES
SITIA
SITIAM
SITIO
SITIOS
SITO
SITOS
SO
SOA
SOAM
SOIS
SOR
SORNICE
SORTE
SORTEIA
SORTES
SORTIDO
SORTIDOS
SOS
SOSIA
SOSIAS
SOU
SUA
SUAM
SUAO
SUE
SUES
SUICA
SUITE
SUITES
SUOR
SURDO
TA
TE
TEIA
TEIAS
TESO
TESOS
TEU
TEUS
TI
TIA
TIAS
TICAO
TIDO
TIDOS
TIM
TINIR
TIO
TIOS
TISICA
TISICAS
TO
TORNE
TORNEI
TOSCA
TOSCAS
UI
URDE
URDIA
URDIAM
USA
USAM
USO
